Vaibhav/day6/Maps.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import "fmt"
+
+func main() {
+
+	// Creating a map
+	// map[key-type]value-type
```

Comment:
take real-time example, so in future we can relate them

Reply:
Okay mahesh.